src/app/component/layout/components/search-popup/search-popup.component.ts

```diff
@@ -45,8 +57,10 @@ export class SearchPopupComponent implements OnInit, OnDestroy {
   }
 
   private getNewsAndTips(news, tips): void {
-    (news && news.length > 0) ? (this.isNewsSearchFound = true, this.newsElements = news) : (this.isNewsSearchFound = false);
-    (tips && tips.length > 0) ? (this.isTipsSearchFound = true, this.tipsElements = tips) : (this.isTipsSearchFound = false);
+    this.isNewsSearchFound = news.length > 0;
+    this.isTipsSearchFound = tips.length > 0;
+    this.newsElements = news.length > 0 ? news : false;
+    this.tipsElements = tips.length > 0 ? tips : false;
```

Comment:
`this.newsElements` and `this.tipsElements` aren't boolean. So, should be something like this
```js
this.newsElements = news.length > 0 ? news : this.newsElements
```

Reply:
fixed